scripts/install-celo.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -euo pipefail
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
+
+# Defaults, can be overwritten by env variables/input parameters
+CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
+NETWORK_DEFAULT="local"
+KEEP_CELO_PASSWORD=${KEEP_CELO_ACCOUNT_PASSWORD:-"password"}
+CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY:-""}
+
+help()
+{
+   echo -e "\nUsage: ENV_VAR(S) $0"\
+           "--config-dir <path>"\
+           "--network <network>"
+   echo -e "\nEnvironment variables:\n"
+   echo -e "\tKEEP_CELO_ACCOUNT_PASSWORD: Unlock an account with a password. Default password is 'password'"
```

Comment:
Same comment as for `install.sh` - it matters only for local, right? If so, let's make it explicit in the description.

Reply:
done